src/components/index/Index.styled.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from "@emotion/styled";
+
+export const Wrapper = styled.div`
+  background-color: #efefef;
+  min-height: 100vh;
```

Comment:
nit: this might not work well on some devices (iphones with notch), one approach which does the same thing but works on all devices is setting the height of the `html, body, #root` element to `100%` and then setting the wrapper height to `100%`

Reply:
Cool, changed!